start.sh

```diff
@@ -595,9 +595,9 @@ if [ "$wallet_backend" == "true" ]; then
 fi
 
 if [ "$block_headers_service" == "true" ]; then
-  save_value 'SPVWALLET_PAYMAIL_BEEF_BLOCK_HEADER_SERVICE_URL' "http://block-headers-service:8080/api/v1/chain/merkleroot/verify"
+  save_value 'SPVWALLET_PAYMAIL_BEEF_BLOCK_HEADERS_SERVICE_URL' "http://block-headers-service:8080/api/v1/chain/merkleroot/verify"
 else
-  save_value 'SPVWALLET_PAYMAIL_BEEF_BLOCK_HEADER_SERVICE_URL' "http://host.docker.internal:8080/api/v1/chain/merkleroot/verify"
+  save_value 'SPVWALLET_PAYMAIL_BEEF_BLOCK_HEADERS_SERVICE_URL' "http://host.docker.internal:8080/api/v1/chain/merkleroot/verify"
```

Comment:
It will be good to announce on channel that everyone need to start fresh ./start.sh configuration to overwrite current BHS urls. 
I mean to not launch it with `./start.sh -l` flag

Reply:
Of course, after merging the changes, I will remind to edit .env.config or re-run ./start.sh without the “-l” flag